app/uk/gov/hmrc/essttpstubs/controllers/EligibilityController.scala

```diff
@@ -47,23 +47,23 @@ class EligibilityController @Inject() (
 
   def retrieveEligibilityData: Action[EligibilityRequest] = Action.async(parse.json[EligibilityRequest]) { implicit request =>
     LoggingHelper.logRequestInfo(logger  = logger, request = request)
-    val maybeResponse: Option[Status] = request.body.idValue match {
-      case "NotFound" => Some(NotFound)
-      case "500Error" => Some(InternalServerError)
-      case "502Error" => Some(BadGateway)
-      case "503Error" => Some(ServiceUnavailable)
-      case "504Error" => Some(GatewayTimeout)
-      case "422Error" => Some(UnprocessableEntity) // de-registered user response from ttp
-      case "499Error" => Some(new Status(499)) // we see some 499's in prod, as far as we can tell they're similar to 502
-      case _          => None
+    val maybeResponse: Option[Status] = request.body.identification.headOption.map(_.idValue.value) match {
```

Comment:
so we don't have to worry about the order of the id's coming in it'd be safer to perhaps see if there's an id that exists which matches one of the things in the pattern match and return the corresponding response if it does